<?xml version="1.0"?>
<launch>
    <node pkg="rostopic" type="rostopic" name="gps_fix_publisher"
          args="pub /gps/fix sensor_msgs/NavSatFix --latch --file=$(find gantry_simulation)/launch/gantry_map.gps"
          output="screen"/>
    <node pkg="satellite_map" type="satellite_tile_downloader.py" name="satellite_tile_downloader"
          output="screen">
        <param name="tile_url" value="https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}"/>
        <param name="zoom_level" value="18"/>
        <param name="num_workers" value="4"/>
        <param name="gps_topic" value="/gps/fix"/>
    </node>

    <node pkg="rviz" type="rviz" name="rviz"
          args="-d $(find gantry_simulation)/launch/gantry_map_satellite.rviz"
          output="screen"/>

</launch>
